oem/ibm/libpldmresponder/file_io_type_pcie.cpp

```diff
@@ -924,10 +937,12 @@ void PCIeInfoHandler::parseTopologyData()
         std::string slot_location_code(slot_location.begin(),
                                        slot_location.end());
 
-        struct SlotLocCodeSuf_t* slot_loc_suf_data =
-            (struct SlotLocCodeSuf_t*)(((uint8_t*)slot_data) + 2 +
-                                       slot_data->slotLocCodesCmnPrtSize);
-        if (slot_loc_suf_data == nullptr)
+        // Each slot location structure contains
+        // { Number of slot location codes (1byte),
+        //   Slot location code Common part size (1byte)}
+        uint8_t* suffix_data = (uint8_t*)slot_data + 2 +
```

Comment:
Please change this magic number 2 too

Reply:
Added the comment to explain why the Magic Number is 2